CONTRIBUTING.md

```
@@ -151,3 +151,39 @@ Apply labels when creating GitHub issues and PRs. Use one from each relevant gro
 | `backend` | Backend only |
 | `android` | Android only |
 | `infra` | CI/CD, GitHub Actions, secrets |
+
+## Issue Template
+
+Use this format when creating GitHub issues:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)